traits/tests/test_get_traits.py

```diff
@@ -17,13 +17,10 @@
 import unittest
 
 from traits.api import (
-    Float,
```

Comment:
unrelated cleanup?

Reply:
Yes; let's move this out of this PR.

Reply:
no, i'm fine with leaving them in. (i don't mind cleaning up the codebase in a piecemeal fashion). i just wanted to confirm that it's unrelated.

Reply:
I do mind. :-) This doesn't really fall under the boy scout rule: it's not an area of the codebase that we're touching otherwise, so it shouldn't be in this PR. PRs should be focused on one thing.